compiler/control/CompileMethod.cpp

```diff
@@ -274,6 +274,7 @@ compileMethodFromDetails(
       TR_Hotness hotness,
       int32_t &rc)
    {
+   TR_VerboseLog::vlogAcquire();
```

Comment:
You do not need to acquire verbose log lock so early on here. There is a lot happening before a writing to verbose log is actually needed. Since there are multiple exit points, you could also consider just using `TR_VerboseLog::writeLineLocked`  instead of writing multiple releases at different exit points.

Reply:
Agreed